buildCompleteWebsite.sh

```diff
@@ -26,31 +26,31 @@ mkdir -p build
 cd build
 if [ -d "xeps" ]; then
 	cd xeps
-	git pull
+	git pull || exit 1;
```

Comment:
I believe you can add `set -e` to the beginning of the script to force and exit for any non-zero result